### DISCUSSION TITLE
Both v3 CSS and JS are not working

### DISCUSSION THREAD

**Discussion by ilovefreesw** [walinejs/waline#2317] [General]
Here both these URLs do not work on this documentation page: https://waline.js.org/en/guide/get-started/#importing-in-html-client

https://unpkg.com/@waline/client@v3/dist/waline.css

https://unpkg.com/@waline/client@v3/dist/waline.js

![image](https://github.com/walinejs/waline/assets/77317006/a3e270a5-56b0-437f-81b7-a61d94c627c4)

**Comment by lizheming** [comment]
it works now~ please try it again

**Reply by ilovefreesw** [reply]
Yes it works now. But it wasn't working before.
